site/src/pages/DeploySettingsPage/GeneralSettingsPage/GeneralSettingsPageView.stories.tsx

```diff
@@ -1,46 +1,67 @@
-import { ComponentMeta, Story } from "@storybook/react"
+import { Meta, StoryObj } from "@storybook/react"
 import { mockApiError, MockDeploymentDAUResponse } from "testHelpers/entities"
-import {
-  GeneralSettingsPageView,
-  GeneralSettingsPageViewProps,
-} from "./GeneralSettingsPageView"
+import { GeneralSettingsPageView } from "./GeneralSettingsPageView"
 
-export default {
+const meta: Meta<typeof GeneralSettingsPageView> = {
   title: "pages/GeneralSettingsPageView",
   component: GeneralSettingsPageView,
   args: {
     deploymentOptions: [
       {
         name: "Access URL",
         description:
-          "External URL to access your deployment. This must be accessible by all provisioned workspaces.",
+          "The URL that users will use to access the Coder deployment.",
```

Comment:
```suggestion
          "The URL users will access the Coder deployment from.",
```

Reply:
This is just a mock 😆 and the text I got from the BE.